cmd/habitatctl/commands/start.go

```diff
@@ -24,20 +24,26 @@ func parseFlags(args []string) ([]string, []string) {
 
 // startCmd represents the start command
 var startCmd = &cobra.Command{
-	Use:                "start [process] [env vars as VAR=value] [flags",
-	Short:              "Starts a habitat process",
-	Long:               `TODO create long description`,
-	FParseErrWhitelist: cobra.FParseErrWhitelist{UnknownFlags: true},
-	DisableFlagParsing: true,
-	Args:               cobra.MinimumNArgs(1),
+	Use:   "start [process] -- [env vars] [flags]",
+	Short: "Starts a habitat process",
+	Long:  `TODO create long description`,
+	Args:  cobra.MinimumNArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
+		process := args[0]
+		fmt.Println("start process ", process)
+		if cmd.ArgsLenAtDash() != 1 && cmd.ArgsLenAtDash() != -1 {
+			fmt.Println("Error: only one process should be specified before -- number specified: ", cmd.ArgsLenAtDash())
+			return
+		}
 		client, err := client.NewClient()
 		if err != nil {
 			fmt.Println("Error: couldn't connect to habitat service")
 			return
 		}
 
 		flags, nonflags := parseFlags(args[1:])
+		fmt.Println("flags ", flags)
```

Comment:
Could we remove the prints just to keep stuff looking clean?

Reply:
oof yeah its a bit messy